spark/v3.3/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestRewritePositionDeleteFiles.java

```diff
@@ -122,8 +123,8 @@ public void testDatePartition() throws Exception {
   @Test
   public void testBooleanPartition() throws Exception {
     createTable("boolean");
-    insertData(i -> i % 2 == 0, 2);
-    testDanglingDelete(2);
+    insertData(PARTITION_COL, i -> i % 2 == 0, 2);
```

Comment:
Thanks, can also you make these methods have a default version that delegates with PARTITION_COL?:
```
  private void insertData(Function<Integer, ?> partitionValueFunction) throws Exception {
    insertData(partitionValueFunction, PARTITION_COL, NUM_DATA_FILES);
  }
```

Reply:
Done. testDanglingDelete also made the same optimization.